### ISSUE TITLE
Failed at node server.js 

### ISSUE THREAD

**Issue by chris-anand** [watson-developer-cloud/food-coach#12]
Hi, I've followed all the steps but while trying to start the app using npm start, it didnt work. 

These are the errors on my command window, I've also attached the debug file. Please let me know what I'm missing here. Thanks in advance.

Windows_NT 6.1.7601
argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\
les\\npm\\bin\\npm-cli.js" "start"
node v6.10.2
npm  v3.10.10
code ELIFECYCLE
food-coach@0.2.1 start: `node server.js`
Exit status 1

Failed at the food-coach@0.2.1 start script 'node server.js'.
Make sure you have the latest version of node.js and npm installed.
If you do, this is most likely a problem with the food-coach package,
not with npm itself.
Tell the author that this fails on your system:
    node server.js
You can get information on how to open an issue for this project with:
    npm bugs food-coach
Or if that isn't available, you can get their info via:
    npm owner ls food-coach
There is likely additional logging output above.

[npm-debug-copy1.txt](https://github.com/watson-developer-cloud/food-coach/files/987020/npm-debug-copy1.txt)

**Comment by ttlew** [comment]
I've also been getting the same error. @stevenpkg can you help with this as well?

**Comment by stevenpkg** [comment]
Will have a look

**Comment by aedocw** [comment]
I hit the same issue, node v7.10.0 on linux.  After npm start failed, I tried the following:

$ node server.js

assert.js:81
  throw new assert.AssertionError({
  ^
AssertionError: url is not valid
    at dbScope (/home/doc/repos/food-coach/node_modules/nano/lib/nano.js:36:10)
    at Object.<anonymous> (/home/doc/repos/food-coach/app.js:197:29)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/home/doc/repos/food-coach/server.js:6:14)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Module.runMain (module.js:605:10)

**Comment by johmca** [comment]
I had the same problem as aedocw above. It seems to be a mismatch between the defaults provided in the .env file and the application code.

If you are not using Cloudant for logging then the app expects there to be no Cloudant URL specified. The default .env file comes as follows

CLOUDANT_URL=<cloudant-url>

The app throws a wobbly becuase it thinks a URL is provided but it is invalid.

When I changed it to

CLOUDANT_URL=

I was able to start the app succesfully with npm start

There probably needs to be a change to the app to not interpret <cloudant-url> as a user provided URL.

I hoep that helps.
